versions/3.8.0/ignore.yaml

```diff
@@ -0,0 +1,12 @@
+tests:
+    - tests.integration.standard.test_metadata.BadMetaTest.test_bad_index
+    - tests.integration.standard.test_metadata.IndexMapTests.test_index_follows_alter
+    - tests.integration.standard.test_metadata.IndexMapTests.test_index_updates
+    - tests.integration.standard.test_metadata.SchemaMetadataTests.test_collection_indexes
+    - tests.integration.standard.test_metadata.SchemaMetadataTests.test_indexes
+    - tests.integration.standard.test_metadata.TestCodeCoverage.test_case_sensitivity
```

Comment:
We have a seperate task to do some cleanups and understand which tests should pass and which aren't supported.
This is just a copy of 3.5.0 ignores for now.